package.json

```diff
@@ -48,6 +48,7 @@
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-hook-form": "^7.51.4",
+    "react-icons": "^5.2.1",
```

Comment:
downloading another icon library for just adding 2 more icons is something, i wont recommend. U can download this svgs separately and import it... thats a btter approach

Reply:
right !

Reply:
will remove it as mention